department-of-veterans-affairs/caseflow
# Dates on download link CSV

### Description
The dates on the downloaded CSV weren't rendering correctly. This PR fixes that.

### Testing Plan
1. Go to View Schedule page
1. Confirm page looks right
1. Confirm downloaded CSV looks right
